static/js/stat_var_hierarchy/stat_var_group_node.tsx

```diff
@@ -246,11 +249,20 @@ export class StatVarGroupNode extends React.Component<
   private fetchData(): void {
     const entityList = this.props.entities;
     this.dataFetchingEntities = this.props.entities;
+    let numEntitiesExistence = this.props.numEntitiesExistence;
+    // When dataSource is specified, the stat var group node info fetch should
+    // also be constrained by the data source. So adding the data source dcid
+    // to the `entities` list for this purpose.
+    const entityDcids = entityList.map((entity) => entity.dcid);
+    if (this.props.dataSource) {
+      entityDcids.push(this.props.dataSource);
+      numEntitiesExistence = entityDcids.length;
```

Comment:
in the case where data source is specified, do all entity dcids have to have data for a stat var in order for that stat var to be displayed in the hierarchy? because in the contained in place examples, we sometimes pass in 50 entities, do all of them need to have data?

Reply:
Yea, this is not optimal right now. Ideally, I think we should separate out the data source and the entities from the constraints. This need the API change though. I would add a todo for this.

Doing this to let the custom DC working for now.